### DISCUSSION TITLE
New User: "Password is Invalid. Please Exclude single quotes[...]"

### DISCUSSION THREAD

**Discussion by Git-Me-Some-Hub** [Security-Onion-Solutions/securityonion#5371] [Unsupported Versions] [answered]
I am having an issues using a randomly generate password for a new user. The password does not have anything that is says I cannot have? IT basically kills the user account and I have to try and delete it. 

Password:  U26Kd9#!DQysrt2&fser4$aQEhY7
![image](https://user-images.githubusercontent.com/17930434/131715447-c372e9c7-a1a9-4ba5-9316-ff5518009597.png)

**Comment by jertel** [comment, marked as answer]
Passwords cannot contain dollar signs. We'll update the error message to include that as well.

**Reply by Git-Me-Some-Hub** [reply]
Great thanks. Is there anyway for it to let you know that before you create the user account?

**Reply by jertel** [reply]
I think we can get an improvement in to make the process less frustrating. Thanks for the suggestion.
